src/js/tsdoc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/tsdoc/v0/tsdoc.schema.json",
+  "extends": ["typedoc/tsdoc.json"],
+  "noStandardTags": false,
+  "tagDefinitions": [
```

Comment:
Can you add a few comments about what these are used for,  for later reference?

Reply:
Well we can't add comments in json files...

Reply:
It shows that I haven't edited json for a while )

Reply:
I'll add a comment in about this file in `jsdoc.py`, because it is indeed easy to forget about it.